library/src/main/java/com/mux/video/upload/internal/TranscoderContext.kt

```diff
@@ -322,24 +314,17 @@ internal class TranscoderContext private constructor(
             videoDecoder!!.flush()
             videoEncoder!!.flush()
             muxVideoFrame()
-            releaseCodecs()
         } catch (err:Exception) {
             logger.e(LOG_TAG, "Failed to standardize input file ${uploadInfo.inputFile}", err)
-        }
-        val duration = System.currentTimeMillis() - started
-        try {
-            muxer!!.stop()
-            muxer!!.release()
+        } finally {
+            releaseCodecs()
```

Comment:
Should `releaseCodecs()` also be called inside the `catch` clause?

Reply:
`finally` is cool this way. It's called after a `try` block, even if an exception or `return` statement would otherwise interrupt control flow

Reply:
Thanks for explaining yeah that is convenient